tools/update_schema.ps1

```diff
@@ -33,23 +35,31 @@ if (-not $isAdmin -and -not $WhatIfPreference) {
     }
 }
 
-$destinations = @(
+$destinationsV1 = @(
     "${env:ProgramFiles(x86)}\IIS Express\config\schema\aspnetcore_schema.xml",
     "${env:ProgramFiles}\IIS Express\config\schema\aspnetcore_schema.xml",
     "${env:windir}\system32\inetsrv\config\schema\aspnetcore_schema.xml"
```

Comment:
Add 32bit w3wp?

Reply:
The 32 bit config section doesn't exist for IIS. I'd assume that 32 bit w3wp uses the config files in the 64 bit location.